tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -1024,7 +1024,7 @@ def _ConstructAndTestSecondGradient(self,
           input_sizes,
           x_init_value=x_init_value,
           delta=1e-2)
-    print("%s second-order gradient error = " % func_name, err)
+    tf_logging.info("%s second-order gradient error = " % func_name, err)
```

Comment:
Should be:
`tf_logging.info("%s second-order gradient error = %.4f" % (func_name, err))`